src/frontend/packages/i18n/package.json

```diff
@@ -6,10 +6,16 @@
     "extract-translation": "yarn extract-translation:desk",
     "extract-translation:desk": "rm -f ./locales/desk/translations-crowdin.json && yarn i18next ../../apps/desk/**/*.{ts,tsx} -c ./i18next-parser.config.mjs -o ./locales/desk/translations-crowdin.json",
     "format-deploy": "yarn format-deploy:desk",
-    "format-deploy:desk": "node ./format-deploy.mjs --app=desk --output=../../apps/desk/src/i18n/translations.json"
+    "format-deploy:desk": "node ./format-deploy.mjs --app=desk --output=../../apps/desk/src/i18n/translations.json",
+    "test": "jest"
   },
   "dependencies": {
+    "@types/jest": "29.5.11",
+    "@types/node": "20.11.6",
     "i18next-parser": "8.8.0",
+    "jest": "29.7.0",
+    "ts-jest": "29.1.2",
+    "typescript": "*",
```

Comment:
I have a question, should we pin this `typescript`? Same question for other package?

Reply:
It is pinned , here: 
https://github.com/numerique-gouv/people/blob/13a58d6fa0b2af52b9b9b4c12053d7eb11cdb528/src/frontend/package.json#L22-L25

To do like that, we have the insurance that all the project is with the same version of certain packages.